assets/src/css/admin/forms.scss

```diff
@@ -7,6 +7,33 @@
  * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
 */
 
+
+.give_options_panel {
```

Comment:
As compared to WordPress default styling for the form field, the input field seems to have more y-axis padding and less x-axis padding.

**Give Core**
![image](https://user-images.githubusercontent.com/1784821/69621195-a0df8a80-1064-11ea-9005-856bbcf31663.png)


**WordPress**
![image](https://user-images.githubusercontent.com/1784821/69621223-ad63e300-1064-11ea-9d80-afaf759e93c8.png)